scripts/SMCodeGen/dataModels/stateMachineAppModel.json

```diff
@@ -291,6 +300,7 @@
                                 "Name":"MB_RESP_FROM_SGW",
                                 "Actions":[
                                     "PROCESS_MB_RESP",
+                                    "SEND_EMM_INFO",
```

Comment:
Send EMM info only after initial attach MBRsp...not all MBRsp.

Reply:
Updated to send after imsi attach.